legal-test-fixture/requirements.txt

```diff
@@ -0,0 +1,66 @@
+# Libraries used by legal_test_fixture
+flask-cors==3.0.8
+xlrd==1.2.0
+xlwt==1.3.0
+# All the things from legal_api (whether they are needed or not)
+Babel==2.7.0
+Click==7.0
+Flask-Babel==0.12.2
+Flask-Migrate==2.5.2
+Flask-Moment==0.9.0
+Flask-SQLAlchemy==2.4.1
+Flask-Script==2.0.6
+Flask==1.1.1
+Jinja2==2.10.3
+Mako==1.1.0
+MarkupSafe==1.1.1
+SQLAlchemy-Continuum==1.3.9
+SQLAlchemy-Utils==0.34.2
+SQLAlchemy==1.3.10
+strict-rfc3339==0.7
+Werkzeug==0.16.0
+alembic==1.2.1
+aniso8601==8.0.0
+asyncio-nats-client==0.9.2
+asyncio-nats-streaming==0.4.0
+attrs==19.1.0
+backoff==1.8.1
+blinker==1.4
+certifi==2019.9.11
+chardet==3.0.4
+datedelta==1.3
+dpath==1.4.2
+ecdsa==0.13.3
+expiringdict==1.1.4
+flask-jwt-oidc==0.1.5
+flask-marshmallow==0.10.1
+flask-restplus==0.13.0
+future==0.18.1
+gunicorn==19.9.0
+idna==2.8
+importlib-metadata==0.23
+itsdangerous==1.1.0
+jsonschema==3.1.1
+launchdarkly-server-sdk==6.10.1
+marshmallow-sqlalchemy==0.19.0
+marshmallow==2.20.5
+more-itertools==7.2.0
+protobuf==3.10.0
+psycopg2-binary==2.8.4
+pyRFC3339==1.1
+pyasn1==0.4.7
+pycountry==19.8.18
+pyrsistent==0.15.4
+python-dateutil==2.8.0
+python-dotenv==0.10.3
+python-editor==1.0.4
+python-jose==3.0.1
+pytz==2019.3
+requests==2.22.0
+rsa==4.0
+semver==2.8.1
+sentry-sdk==0.13.0
+six==1.12.0
+urllib3==1.25.6
+zipp==0.6.0
+git+https://github.com/bcgov/lear.git#egg=registry_schemas&subdirectory=schemas
```

Comment:
a large group of these aren't used, but loaded anyways. please pare down the loaded packages.

Reply:
Essential cleanup work! Resolving on new PR (rebasing is fubared)